gremlin-console/src/main/java/org/apache/tinkerpop/gremlin/console/jsr223/DriverRemoteAcceptor.java

```diff
@@ -54,6 +54,7 @@
  */
 public class DriverRemoteAcceptor implements RemoteAcceptor {
     public static final int NO_TIMEOUT = 0;
+    public static final String USER_AGENT = "Gremlin Console";
```

Comment:
I don't want to get too fancy, but perhaps it would be better to more closely follow the User-Agent standard from [rfc2616](https://www.ietf.org/rfc/rfc2616.txt) which calls for:

```text
"User-Agent" ":" 1*( product | comment )
```

where "product" is just:

```text
product         = token ["/" product-version]
product-version = token
```

For our purposes, I think it would be nice if we did:

```text
Gremlin Console/<version>
```

You can get the version from: 

https://github.com/apache/tinkerpop/blob/2ba15943ff276167432a65ca5a6fc043c873492d/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/util/Gremlin.java#L38